README.md

```diff
@@ -1412,6 +1412,7 @@ _Related: [Monitoring](#monitoring)_
 - [Dashboard](https://github.com/phntxx/dashboard) - Minimalist homepage for organizing your web applications and bookmarks using JSON-files. `MIT` `Nodejs/Docker`
 - [DashMachine](https://github.com/rmountjoy92/DashMachine) - Another web application bookmark dashboard, with fun features. `GPL-3.0` `Python`
 - [Dashy](https://github.com/lissy93/dashy) - Feature-rich homepage for your homelab, with easy YAML configuration. ([Demo](https://demo.dashy.to/)) `MIT` `Nodejs/Docker`
+- [Fenrus](https://github.com/revenz/fenrus) - A self hosted personal home page that allows for multiple users, guest access and multiple dashboards for each user.  It also has "Smart Apps" which displays live data for those apps. `GPL-3.0` `Nodejs`
```

Comment:
- double space `user.  It` -> `user. It`
- typo `which displays` -> `which display`